_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,120 @@ Parameter | Description | Type | Required
+                                        "terms": {
+                                            "source_field": "day_of_week",
+                                            "target_field": "day_of_week"
+                                        }
+                                    }
+                                ],
+                                "metrics": [
+                                    {
+                                        "source_field": "taxless_total_price",
+                                        "metrics": [
+                                            {
+                                                "sum": {}
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "source_field": "total_quantity",
+                                        "metrics": [
+                                            {
+                                                "avg": {}
+                                            },
+                                            {
+                                                "max": {}
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                ],
+                "transitions": []
+            }
+        ]
+    }
+}
+````
+
+#### Field parameters
+
+The following field parameters are required:
+
+Field Parameter | Data Type | Description 
+:--- | :--- |:---
+description | String | The description name of the ISM policy. This field is optional.
+default_state | String | The default state to run for the ISM policy. Set the description to `rollup` for the ISM rollup policy.
+states | Nested list of objects | Define the states to be used in the ISM rollup policy.
+states.name | Nested list of objects | The description name of the state within the ISM policy. In this case, you will need to match the `default_state` description of "`rollup`."
+states.actions | Nested list of objects | Define the actions to be run by the state.
+states.transition | Nested list of objects | Define the conditions required to move on to the next state. When left blank, the policy completes its management of the index.
+states.actions.rollup | Nested list of objects | Define the actions taken during index rollup.
+states.actions.ism_rollup | Nested list of objects | Define the actions taken during ISM rollup.
+states.actions.rollup.ism_rollup.description | String | The description name of the `ism_rollup` field. This field is optional.
+states.actions.rollup.ism_rollup.target_index | String | The index to be used by the ISM rollup profile.
+states.actions.rollup.ism_rollup.page_size | Number | The number of indexes to be returned per page.
```

Comment:
Is this the number of indexes or the number of documents per bucket?

Reply:
good catch! I'll get that updated